gtsfm/scene_optimizer.py

```diff
@@ -443,6 +465,18 @@ def save_gtsfm_data(
                 save_dir=os.path.join(output_dir, "ba_output"),
             )
         )
+
+        # Save the ground truth in the same format, for visualization.
+        # We use the estimated tracks here, with ground truth camera poses.
+        gt_gtsfm_data = dask.delayed(get_gt_gtsfm_data)(cameras_gt, ba_output_graph)
+        saving_graph_list.append(
+            dask.delayed(io_utils.export_model_as_colmap_text)(
+                gt_gtsfm_data,
+                image_graph,
```

Comment:
if possible, would be nice to have all 3 as keyword args, `gtsfm_data=gt_gtsfm_data, images=image_graph, ...`

https://github.com/borglab/gtsfm/blob/master/gtsfm/utils/io.py#L163-L165